staging/src/k8s.io/api/discovery/v1/types.go

```diff
@@ -72,8 +72,9 @@ type Endpoint struct {
 	// addresses of this endpoint. The contents of this field are interpreted
 	// according to the corresponding EndpointSlice addressType field. Consumers
 	// must handle different types of addresses in the context of their own
-	// capabilities. This must contain at least one address but no more than
-	// 100.
+	// capabilities. This must contain at least one address but no more than 100.
+	// these are all assumed to be fungible and clients may choose to only use the first element.
+	// refer to: https://github.com/kubernetes/kubernetes/issues/106267
```

Comment:
How about `https://issue.k8s.io/106267` as the URL?

Reply:
it's better :)